src/contexts/NodesContext.tsx

```diff
@@ -11,11 +11,13 @@ import { EntityType } from "./AppContext";
 type NodesContextType = {
   nodes: Node[] | [];
   setNodes: (nodes: Node[] | []) => void;
+  globalStats: { medianTTFB: number };
```

Comment:
I think it would be cleaner to add:
```
type GlobalStats = {
medianTTFB: number,
... 
};

type NodesContextType = {
  nodes: Node[] | [];
  setNodes: (nodes: Node[] | []) => void;
  globalStats: GlobalStats
};

....

const [globalStats, setGlobalStats] = useState<GlobalStats>(initialValues.globalStats);
```